src/applications/facility-locator/containers/FacilitesMap.jsx

```diff
@@ -0,0 +1,469 @@
+import {
+  facilitiesPpmsSuppressCommunityCare,
+  facilitiesPpmsSuppressPharmacies,
+  facilityLocatorPredictiveLocationSearch,
+} from '../utils/selectors';
+import ResultsList from '../components/ResultsList';
+import PaginationWrapper from '../components/PaginationWrapper';
+import SearchControls from '../components/SearchControls';
+import SearchResultsHeader from '../components/SearchResultsHeader';
+import { browserHistory } from 'react-router';
+import vaDebounce from 'platform/utilities/data/debounce';
+import { setFocus, clearLocationMarkers, buildMarker } from '../utils/helpers';
+import { MARKER_LETTERS } from '../constants';
+import { distBetween } from '../utils/facilityDistance';
+import { isEmpty } from 'lodash';
+import { Tab, TabList, TabPanel, Tabs } from 'react-tabs';
+import SearchResult from '../components/SearchResult';
+import { recordZoomEvent, recordPanEvent } from '../utils/analytics';
+import { otherToolsLink, coronavirusUpdate } from '../utils/mapLinks';
+
+let currentZoom = 3;
+
+const FacilitiesMap = props => {
+  const [map, setMap] = useState(null);
+  const searchResultTitleRef = useRef(null);
+  const [isMobile, setIsMobile] = useState(window.innerWidth <= 481);
+  const [isSearching, setIsSearching] = useState(false);
+
+  const syncStateWithLocation = location => {
+    if (
+      location.query.address &&
+      props.currentQuery.searchString !== location.query.address &&
+      !props.currentQuery.inProgress
+    ) {
+      props.genBBoxFromAddress({
+        searchString: location.query.address,
+        context: location.query.context,
+      });
+    }
+  };
+
+  /**
+   * Search when the component renders with a sharable url
+   */
+  const searchWithUrl = () => {
+    // Check for scenario when results are in the store
+    if (!!props.location.search && props.results && props.results.length > 0) {
+      return;
+    }
+    const { location } = props;
+
+    if (!isEmpty(location.query)) {
+      props.updateSearchQuery({
+        facilityType: location.query.facilityType,
+        serviceType: location.query.serviceType,
+      });
+    }
+
+    if (location.query.address) {
+      props.genBBoxFromAddress({
+        searchString: location.query.address,
+        context: location.query.context,
+      });
+      setIsSearching(true);
+    }
+  };
+
+  useEffect(() => {
+    const listener = browserHistory.listen(location => {
+      syncStateWithLocation(location);
+    });
+
+    const setMobile = () => {
+      setIsMobile(window.innerWidth <= 481);
+    };
+
+    searchWithUrl();
+
+    const debouncedResize = vaDebounce(250, setMobile);
+    window.addEventListener('resize', debouncedResize);
+    return () => {
+      listener();
+      window.removeEventListener('resize', debouncedResize);
+    };
+  }, []);
+
+  const updateUrlParams = params => {
+    const { location, currentQuery } = props;
+
+    const queryParams = {
+      ...location.query,
+      zoomLevel: currentQuery.zoomLevel,
```

Comment:
Do we still need a `zoomLevel` in the query? I thought we were using fitBounds to determine the zoom level.